src/scss/components/optional/_badges.scss

```diff
@@ -77,6 +83,21 @@ $badge-angled-radius        : 2px;
             color: $badge-award-color;
         }
 
+        .c-badge--ok {
```

Comment:
May be worth adding a comment as to what this modifier is – `ok` is a bit vague as to the type (compared to `error` etc).  Is it used in combination with messsages?

Reply:
It's only used on the item quantity indicator on menus, see attached image. I wasn't really sure what else to call it but as it's green I went with OK 😸 

![Screenshot 2019-04-11 at 12 31 21](https://user-images.githubusercontent.com/89059/55954649-27bb3080-5c57-11e9-9e65-2096705e4c78.png)

Reply:
Any thoughts on what else it could be called?

Reply:
I have renamed to `c-badge--success--alt` and added a description.

Reply:
Sorry for delay in replying – I was away Thurs/Fri last week

That name looks good though 👍